eventsources/sources/bitbucket/basic_auth_strategy.go

```diff
@@ -0,0 +1,51 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bitbucket
+
+import (
+	bitbucketv2 "github.com/ktrysmt/go-bitbucket"
```

Comment:
lets run go import to organize imports for all files

Reply:
Hmm not sure I follow.. seems pretty organized to me, or am I missing something?

Reply:
The go imports will group core libs and external libs into 2 sections, right now the imports have unnecessary extra lines between them. Not important but just housekeeping stuff.